Quotient/room.h

```diff
@@ -235,6 +238,12 @@ class QUOTIENT_API Room : public QObject {
     // Property accessors
 
     Connection* connection() const;
+
+    /**
+     * @brief Get a RoomMember object for local user.
+     */
```

Comment:
```suggestion
    //! Get a RoomMember object for local user.
```
We try using Doxygen style instead of Javadoc style. And `brief` is not needed for one-liners.